### PR TITLE
Add fully testcase to A1cCalculator

### PR DESCRIPTION
**Pull request by raacker**
I already tried to write all testcase but I had no idea how to test remaining methods. 

Finished it

### CODE REVIEW

File: app/src/test/java/org/glucosio/android/presenter/A1CCalculatorPresenterTest.java [hunk 1]
```diff
@@ -66,4 +72,24 @@ public void ShouldCalculatable_WhenUserPreferredPercentage() throws Exception {
...
+    }
+
+    @Test
+    public void ShouldReturnCorrectA1CUnit_WhenGetterCalled() throws Exception {
+        when(userMock.getPreferred_unit_a1c()).thenReturn("mmol/mol");
+        assertThat(presenter.getA1cUnit()).isEqualTo("mmol/mol");
+    }
+
+    @Test
+    public void ShouldSaveA1CCorrectly_WhenUserPreferredMmolForA1C() throws Exception {
```
Comment (emartynov) [inline app/src/test/java/org/glucosio/android/presenter/A1CCalculatorPresenterTest.java]: Better naming probably - should finish activity after saving

Comment (raacker) [inline app/src/test/java/org/glucosio/android/presenter/A1CCalculatorPresenterTest.java]: right! my mistake. thank you for pointing out

**Comment by emartynov** [comment]
Could you also remove unused imports?

**Comment by raacker** [comment]
eeh.. yeah so much issues from codacy :) Okay!

**Comment by emartynov** [comment]
Thanks! Why do you rewrite history all the time? Why not just add new commits to pull request?

**Comment by raacker** [comment]
Ah because just keep adding tons of commits with small changes make total history dirty! Maybe make new commit all the time and squash them all after approved, it would be best case. But just for case. Make commit simple and valuable is my commit style

**Comment by emartynov** [comment]
There are no tons of commits. And we usually squash all changes in one commit per PR. 

You current style with one commit and history rewriting makes harder to me follow the changes.

**Comment by emartynov** [comment]
Thanks! Merging!

**Comment by raacker** [comment]
Okay then. I'll make all commit for each changes :)   thank you!

**Pull request merged**